### PR TITLE
Riscv64 port

### LINKED ISSUE #3: RISCV support
Nice work. Any plan to support riscv ?

### PR DESCRIPTION
**Pull request by Skallwar**
Closes #3

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -41,16 +41,31 @@
 //! The QEMU binary will execute `exit((arg << 1) | 1)`. The is hardcoded in the QEMU sources.
 //! Therefore, with `isa-debug-exit`, it is not possible to let QEMU invoke `exit(0)`.
 //!
+//! # RISCV64
```
Comment (andre-richter) [inline src/lib.rs:44]: Can you put it between the ARM and x86 (trying to be alphabetical here).

Also, please put a copy of this into the the top level README.md.

File: src/lib.rs [hunk 2]
```diff
@@ -64,3 +79,6 @@ mod host_stubs;
 
 #[cfg(target_arch = "x86_64")]
 pub use host_stubs::*;
+
+#[cfg(target_arch = "riscv64")]
```
Comment (andre-richter) [inline src/lib.rs:83]: Can you put it between the ARM and x86 (trying to be alphabetical here).

File: src/lib.rs [hunk 3]
```diff
@@ -64,3 +79,6 @@ mod host_stubs;
...
+pub mod aarch64;
```
Comment (andre-richter) [inline src/lib.rs:84]:
```suggestion
pub mod riscv64;
```

**Review by andre-richter — CHANGES_REQUESTED** [review]
Awesome stuff, thanks!
Reminds me I need to port the aarch64 to the new asm! macro.

I left a few comments, please see if you can address them.

File: src/lib.rs (hunk 1 above)
Comment (Skallwar) [reply src/lib.rs]: Fixed

File: src/lib.rs (hunk 2 above)
Comment (Skallwar) [reply src/lib.rs]: Fixed

File: src/lib.rs (hunk 3 above)
Comment (Skallwar) [reply src/lib.rs]: Nice catch

**Comment by andre-richter** [comment]
Published on crates.io as version `1.4`.

Thanks again.

**Comment by Skallwar** [comment]
This might also work with RISCV32. I will check this tomorrow

**Pull request merged**